{
    "title": "An Out of Bounds flaw was found fig2dev version 3.2.8a. A flawed bounds check in read_objects() could allow an attacker to provide a crafted malicious input causing the application to either crash or in some cases cause memory corruption. The highest threat from this vulnerability is to integrity as well as system availability.",
    "score": 7.1,
    "severity": "HIGH",
    "cwe_id": "CWE-787",
    "cvss_vector": "CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:H/A:H",
    "cpe_match": [
        "cpe:2.3:a:fig2dev_project:fig2dev:3.2.8:a:*:*:*:*:*:*",
        "cpe:2.3:o:fedoraproject:fedora:33:*:*:*:*:*:*:*",
        "cpe:2.3:o:fedoraproject:fedora:34:*:*:*:*:*:*:*",
        "cpe:2.3:o:debian:debian_linux:9.0:*:*:*:*:*:*:*"
    ],
    "cve_source": [
        "NVD"
    ],
    "ttpMapping": [
        {
            "techniqueId": "T1033",
            "confidence": "11.2",
            "techniqueName": "System Owner/User Discovery"
        }
    ]
}
